src/sentry/sentry_metrics/querying/metadata/metric_spans.py

```diff
@@ -157,7 +174,7 @@ def _get_spans(
     ) -> Sequence[Span]:
         span_ids = self._get_span_ids_from_metrics_summaries(
             metric_mri=metric_mri,
-            conditions=conditions,
+            conditions=transform_to_tags(conditions),
```

Comment:
Decided to move the transformations in each `*SpansSource` since the needed transformations differ.